crypto-lib/src/main/kotlin/ee/ria/DigiDoc/cryptolib/CryptoContainer.kt

```diff
@@ -128,33 +122,29 @@ class CryptoContainer
         ): File? {
             val file = sanitizeString(dataFile.name, "")?.let { File(directory, it) }
```

Comment:
```suggestion
            val tmp =  sanitizeString(dataFile.name, "")?: return
            val file = File(directory, tmp)
```
Or fix the sanitizeString not to return null